src/class/api.ga4gh.php

```diff
@@ -668,21 +716,19 @@ private function showTableData ($sTableName)
     private function showTableDataPage ($sTableName, $aPage)
     {
         // Shows table data page.
-        global $_CONF, $_SETT;
+        global $_SETT;
 
         if ($sTableName == 'variants') {
             list(, $sBuild, $sChr, $sPosition) = array_pad($aPage, 4, '1');
 
-            if ($sBuild != $_CONF['refseq_build']) {
-                // We don't support this yet, because we can't use an index on a
-                //  search on the other genome build. Wait for LOVD 3.1.
+            if (!isset($this->aActiveGBs[$sBuild])) {
+                // We can only support searches on genome builds that are
+                //  active in the database.
                 $this->API->nHTTPStatus = 400; // Send 400 Bad Request.
-                if (!isset($_SETT['human_builds'][$sBuild])) {
-                    $this->API->aResponse = array('errors' => array('title' => 'Unrecognized genome build.'));
-                } else {
-                    $this->API->aResponse = array('errors' => array('title' => 'Unsupported genome build.'));
-                }
-                $this->API->aResponse['errors']['detail'] = 'We can not use genome build ' . $sBuild . '. Please choose from: \'' . $_CONF['refseq_build'] . '\'.';
+                $this->API->aResponse = array('errors' => array('title' =>
+                    (isset($_SETT['human_builds'][$sBuild])? 'Unsupported' : 'Unrecognized')  . ' genome build.')
+                );
+                $this->API->aResponse['errors']['detail'] = 'We cannot use genome build ' . $sBuild . '. Please choose from: \'' . implode('|', array_keys($this->aActiveGBs)) . '\'.';
```

Comment:
This will look a bit weird;
```
Please choose from: 'hg19'.
```
looks OK, but 
```
Please choose from: 'hg19|hg38'.
```
Looks a bit weird. Probably `'hg19', 'hg38'` would be more logical for humans.

Reply:
Done as recommended!